src/__init__.py

```diff
@@ -5,30 +5,42 @@
 from dotenv import load_dotenv
 
 from src.models import db
-from src.user import usr
-from src.config.config import TestingConfig
+from src.google import google_bp
+from src.home import home
+from src.manage import create_tables
 
 load_dotenv()
 
 def create_app(test_config=None):
     app: Flask = Flask(__name__, instance_relative_config=True)
     
     if test_config is None:
+        
+        # Heroku Postgresql hack.
+        db_url = str(os.environ.get('DATABASE_URL'))
+        if db_url.startswith('postgres://'):
+            db_url = db_url.replace('postgres://', 'postgresql://')
+            
         app.config.from_mapping(
             SECRET_KEY=os.environ.get('SECRET_KEY'),
```

Comment:
Just a note, you also want to get this value and check whether it is not empty before passing it to the variable SECRET_KEY. just so you can catch errors early and return a custom error response.